### PR TITLE
Article on "Invert Binary Tree" Added & sidebar.js updated

### LINKED ISSUE #336: Article on Inversion of Binary Tree
## 💥 Proposal

An inversion, or mirror, of a Binary Tree ​ is just a Binary Tree  whose left and right children (of all non-leaf nodes) are swapped.

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

Yes

### PR DESCRIPTION
**Pull request by Nikitha2309**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes :)

### Description

Given a Binary Tree as input, output its Inverted Binary Tree

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Fixes #336

### CODE REVIEW

**Comment by Nikitha2309** [comment]
@HarshCasper @gat786 
Pls review the PR and let me know if there are any changes to be made 
Thank you 🙂

**Comment by gat786** [comment]
You can improve on the title of this pr. I would suggest writing the article's name which you have contributed about. It works if the information about sidebar.js is not mentioned we consider it as a taken that you have to update it everytime anyways.

File: docs/data-structures/invert-binary-tree.mdx
Comment (gat786) [inline docs/data-structures/invert-binary-tree.mdx:17]: is  it necessary to provide the image as a blob here? Can you simply use a image url instead?

**Comment by gat786** [comment]
Rest everything looks good and passes my quality check barrier.

File: docs/data-structures/invert-binary-tree.mdx
Comment (Nikitha2309) [reply docs/data-structures/invert-binary-tree.mdx:17]: > is it necessary to provide the image as a blob here? Can you simply use a image url instead?

Yes i think. Because when this is the URL of the image when I tried to copy :)

**Comment by Nikitha2309** [comment]
> You can improve on the title of this pr. I would suggest writing the article's name which you have contributed about. It works if the information about sidebar.js is not mentioned we consider it as a taken that you have to update it everytime anyways.

Yeah, sure.

**Comment by Nikitha2309** [comment]
@gat786 
so will u merge the code?
Is there anything else I have to do? 🙂

**Comment by gat786** [comment]
I dont know any project specific details so merging it without the actual maintainers notice would be a bad idea. So let harsh see it once. I have already approved it so he will merge it if there are no big things you are missing out on.

**Comment by Nikitha2309** [comment]
> I dont know any project specific details so merging it without the actual maintainers notice would be a bad idea. So let harsh see it once. I have already approved it so he will merge it if there are no big things you are missing out on.

Ok 🙂. Thanks for the approval.

**Comment by Nikitha2309** [comment]
@HarshCasper @ankitaggarwal23 
I have resolved the conflicts 3 times so far , due to a new merge in sidebar.js
I think its better to resolve it , before u merge the PR ,instead of resolving them again and again.
Hence please tag me if u review this PR . I will be available to resolve the conflicts.
Thank you

File: docs/data-structures/invert-binary-tree.mdx
Comment (HarshCasper) [inline docs/data-structures/invert-binary-tree.mdx:17]: Please use an mage URL here instead of a Blob

File: docs/data-structures/invert-binary-tree.mdx
Comment (HarshCasper) [inline docs/data-structures/invert-binary-tree.mdx:213]: Give a space after heading

File: docs/data-structures/invert-binary-tree.mdx
Comment (HarshCasper) [inline docs/data-structures/invert-binary-tree.mdx:190]: Remove this line

File: docs/data-structures/invert-binary-tree.mdx
Comment (HarshCasper) [inline docs/data-structures/invert-binary-tree.mdx:8]: Can you add a more intuitive introduction here?

File: docs/data-structures/invert-binary-tree.mdx
Comment (HarshCasper) [inline docs/data-structures/invert-binary-tree.mdx:12]: The explanation isn't much explainable. Can you add a better explanation, with aid of text and pictures here?

File: docs/data-structures/invert-binary-tree.mdx
Comment (HarshCasper) [inline docs/data-structures/invert-binary-tree.mdx:15]: Keep it `Examples`

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Review by ankitaggarwal23 — COMMENTED** [review]
Closing this due to inactivity since last 21 days. Thank you for your valuable time.😊

**Pull request closed without merging**
